args.go

```diff
@@ -44,6 +46,9 @@ func parseArgs() (common.Config, error) {
 			PlaceHolder("file.csv").
 			Short('o').
 			String(),
+		CPUs: app.Flag("cpus", "Maximum number of logical CPUs").
```

Comment:
Descriptions of all other flags end with a full stop. One extra sentence could be added: `Defaults to the number of available CPU threads (or the GOMAXPROCS env variable).`